packages/react/src/utils/scriptLoader.ts

```diff
@@ -30,18 +31,30 @@ const forceStagingReleaseForClerkFapi = (frontendApi: string): boolean => {
   );
 };
 
-function getScriptSrc({ frontendApi, scriptUrl, scriptVariant = '' }: LoadScriptParams): string {
+function getScriptSrc({ publishableKey, frontendApi, scriptUrl, scriptVariant = '' }: LoadScriptParams): string {
   if (scriptUrl) {
     return scriptUrl;
   }
 
+  let scriptHost: string;
+  if (publishableKey) {
+    const { frontendApi } = parsePublishableKey(publishableKey)!;
+    scriptHost = frontendApi;
+  } else if (frontendApi) {
+    scriptHost = frontendApi;
+  } else {
+    throw new Error(
+      'Internal Clerk Error: Neither frontendApi nor publishableKey passed to getScriptSrc. Please report to support@clerk.dev if you are seeing this while developing an application with Clerk. (package=@clerk/clerk-react)',
```

Comment:
🔧 We should throw this error from the errorThrower. The message should be `Missing frontendApi or publishableKey`.

Reply:
Introduced a new `MissingFrontendApiPublishableKeyErrorMessage` in our errorThrower and used this here instead of the hardcoded error that we were using before